### DISCUSSION TITLE
How to open files in FileUpload temporarily

### DISCUSSION THREAD

**Discussion by lucacastelnuovo** [filamentphp/filament#6781] [Help] [answered]
Here https://filamentphp.com/docs/2.x/forms/fields#file-upload it is written that uploaded files can be viewed by setting the `->enableOpen()` method. 

This unfortunately does not work for private s3 buckets. Is there a way to open the file using the `temporaryUrl()` method?

**Comment by danharrin** [comment, marked as answer]
Does it work if you use `visibility('privtae')` ?

**Reply by lucacastelnuovo** [reply]
Yes, that worked. Thanks a lot for responding so quickly.

**Reply by lucacastelnuovo** [reply]
Small note for other copy pasting this solution, use `visibility('private')`
